Forms/Modify-the-widget-annotation-in-PDF-document/.NET/Modify-the-widget-annotation-in-PDF-document/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Syncfusion.Pdf.Interactive;
+using Syncfusion.Pdf.Parsing;
+using Syncfusion.Pdf;
+
+//Load the PDF document.
+FileStream docStream = new FileStream(Path.GetFullPath(@"Data/Input.pdf"), FileMode.Open, FileAccess.Read);
+PdfLoadedDocument loadedDocument = new PdfLoadedDocument(docStream);
+// Loop through each page in the loaded PDF document.
+foreach (PdfLoadedPage page in loadedDocument.Pages)
+{
+    // Access the form within the loaded document.
+    PdfLoadedForm form = loadedDocument.Form;
+    // Check if the forms are null.
+    if (form == null)
```

Comment:
Whether it is required to check whether form is null. It is not needed. The unlinked annotation will be added to the collection. So remove this condition.

Reply:
Removed